.circleci/config.yml

```diff
@@ -219,3 +253,13 @@ workflows:
           name: "SPM Ubuntu build"
       - example-app-build:
           name: "Build example app"
+      - approve-publish-documentation:
+          name: "Approve Publish Documentation"
```

Comment:
Just as an idea: 
* We can build docs for every commit and store it [in artifacts](https://github.com/mapbox/mapbox-navigation-ios/blob/fc37d6661e7ef73959e0a30108f00b0e487e281b/.circleci/config.yml#L648-L650)
* Then there is an approval job for publishing (that works is for tags only)
* And once approved we can push docs to `publisher-production`